### PR TITLE
Security updates for NPM

### PR DESCRIPTION
**Pull request by BNewing**
This updates the Composer packages and NPM modules, to help improve security, as part of our regular updates.

### CODE REVIEW

File: web/app/themes/npm/package.json [hunk 1]
```diff
@@ -29,16 +29,15 @@
     },
     "dependencies": {
         "bootstrap": "^3.4.1",
-        "bootstrap-dropdown-replacement": "^1.0.7",
         "jquery": "^3.4.0"
     },
     "devDependencies": {
-        "cross-env": "^5.2.0",
+        "cross-env": "^7.0.2",
```
Comment (wilson1000-MoJ) [inline web/app/themes/npm/package.json]: Hi. It might worth noting that 'cross-env' isn't used in this site.

**Review by wilson1000-MoJ — APPROVED** [review]
All looks good.

File: web/app/themes/npm/package.json (hunk 1 above)
Comment (BNewing) [reply web/app/themes/npm/package.json]: Ok thanks :) I'll remove it, test it, push up that commit then merge

Comment (BNewing) [reply web/app/themes/npm/package.json]: Now removed, so I'll merge

**Pull request merged**